examples/stable_diffusion_v2/README.md

```diff
@@ -129,18 +129,42 @@ DreamBooth allows users to generate contextualized images of one subject using j
 
 Please refer to the tutorial of [DreamBooth for Stable Diffusion Finetuning](dreambooth_finetune.md)
 
-### 5. Evaluation
+### 5. CN-CLIP
+
+[CN-CLIP](https://github.com/ofa-sys/chineseclip) is an open-source CLIP implementation that is trained on an extensive dataset on Chinese text, image pairs. The difference is mostly on tokenizers and the first embedding layer. We support replacing the original CLIP weights with CN-Clip weights using the arg `--custom_text_encoder`.
+
+To use CN-Clip in your training or inference,
+
+First download our provided weights [ViT-H/14](https://download.mindspore.cn/toolkits/mindone/stable_diffusion/ms_cnclip_h14-d780480a.ckpt) converted from torch.
+
+Then use the predefined *cnclip.yaml files in `train_config.json`:
+
+`# "model_config": "configs/v2-train-cnclip.yaml"`
+
+Finally modify a bit to your old train / infer commands:
+
+```
+# training:
+python train_text_to_image.py --custom_text_encoder PATH_TO_CNCLIP.ckpt ...
+
+# inference: modify args passed to text_to_image.py
+python text_to_image.py --config configs/v2-inference-cnclip.yaml ...
+```
+
+### 6. Evaluation
 
 Please refer to [Evaluation for Diffusion Models](eval/README.md)
 
 - - -
 ## Stable Diffusion 1.x
 
+We provide [SD1.5 weights](https://download.mindspore.cn/toolkits/mindone/stable_diffusion/ms_v1_5_pruned_emaonly-d0ab7146.ckpt) converted from torch to MindSpore.
+
 ### 1. Chinese Text-to-Image Generation
```

Comment:
The sub-title should be updated too. 

## Stable Diffusion 1.5

### English Text-to-Image

### Chinese Text-to-Image

...